src/components/layout/yearInReview/2023/leadership.tsx

```diff
@@ -0,0 +1,72 @@
+
+import { DarkButton } from 'components/decoration/buttons';
+import { SectionHeader } from 'components/decoration/textBlocks';
+import CustomImage from 'components/decoration/customImage';
+
+const Leadership: React.FC = () => {
+  const intl = useIntl();
+
+  const buttonLabel = intl.formatMessage({
+    id: 'page.year-in-review.2023.section.leadership.label',
+    defaultMessage: 'Read the Announcement',
+  });
+
+  return (
+    <>
+      <div className='flex flex-col md:flex-row text-left bg-white'>
+        <div className='basis-full md:basis-1/2 flex justify-center md:justify-end pt-10 md:py-20 px-5 md:px-10'>
+          <div className='flex-grow max-w-3xl flex justify-center'>
+            <div>
+              <SectionHeader
+                header={intl.formatMessage({
+                  id: 'page.year-in-review.2023.section.leadership.heading',
+                  defaultMessage: 'New horizons in <b>OUR LEADERSHIP</b>',
+                })}
+                stackEntries
+              />
+              <div className='text-lg pt-8 pb-4'>
+                <FormattedMessage
+                  id='page.year-in-review.2023.section.leadership.content'
+                  defaultMessage="Mid-summer this year, we decided to refine the way we build capacity with changes in our leadership. <no-localization>David van Beveren</no-localization>, who founded <no-localization>Vegan Hacktivists</no-localization> in 2019, transitioned to the role of Founder and President, now overseeing the organization's long-term vision and broader impact.{break}{break}Under <no-localization>David’s</no-localization> guidance <no-localization>Vegan Hacktivists</no-localization> grew significantly: what started as a one-person team turned into an organization of over 50 active volunteers, 4 full-time employees, 8 stipend volunteers, and 2,500 volunteers within our <no-localization>Playground</no-localization> network. Since the transition, <no-localization>David</no-localization> has been focusing on the overall direction and strategy, building partnerships and representing the organization in various events and capacities.{break}{break}On the other side of the exchange, <no-localization>James Morgan</no-localization> stepped up as our new Executive Director, bringing his commitment to the cause and expertise in product management to the forefront of our operations. Since joining <no-localization>VH</no-localization>, <no-localization>James</no-localization> has played a pivotal role in enhancing our processes and supporting our growth, originally starting as a volunteer and quickly becoming indispensable as our Director of Operations. He helped lay the foundation of what <no-localization>VH</no-localization> is today, and now, both of our leaders continue to work in synergy to advance our mission and support the movement with innovative tech solutions, strengthening advocacy for animals worldwide.{break}{break}<b>First time hearing the news?</b>"
```

Comment:
Please ensure that placeholders like {break} are wrapped in no-localization tags <no-localization>{break}</no-localization> to ensure they are ignored when auto translated